utils/build/docker/python/python3.12.base.Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN python --version && curl --version
 # install python deps
 ENV PIP_ROOT_USER_ACTION=ignore
 RUN pip install --upgrade pip
-RUN pip install django pycryptodome gunicorn gevent requests boto3==1.34.141 'moto[ec2,s3,all]'==5.0.14
```

Comment:
I would advocate for keeping this one : it'll check python 3.12, where the other one check python3.11

Reply:
ah, interesting. ok, i'll rearrange things

Reply:
ok, they're back, i think!